### ISSUE TITLE
Add React 19 as peer dependency

### ISSUE THREAD

**Issue by Jaival** [linkstrifer/react-rough-notation#33]
**Is your feature request related to a problem? Please describe.**
Installing react-rough-notion with react 19 gives peer dependency errors.

**Describe the solution you'd like**
Adding React 19 to peer dependencies will solve this issue.

**Comment by TheAlgoTradingResources-Cloud** [comment]
any update here?

**Comment by Jaival** [comment]
No, there is no answer; I reached out to the author on Twitter but no answer.

**Comment by Rudrajiii** [comment]
Hey comeon man, i wanna add this shit in my damnnn portfoliooooo 
😭

**Comment by Jaival** [comment]
You can still use the package by using bun/pnpn or  npm install --legacy-peer-deps.

**Comment by Rudrajiii** [comment]
> You can still use the package by using bun/pnpn or npm install --legacy-peer-deps.

![Image](https://github.com/user-attachments/assets/dc826a5d-9b3e-4ff4-a126-69df75ab576a)
😭

**Comment by Jaival** [comment]
Could you upload your project to sandbox and share it with me?

**Comment by linkstrifer** [comment]
This is being tested in v1.0.7 (beta), let me know if it don't work
